config/webhook/service.yaml

```diff
@@ -2,8 +2,7 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: webhook-service
-  namespace: system
+  name: storageos-webhook-service
```

Comment:
It's just `storageos-webhook` now :slightly_smiling_face:

Reply:
Grr...  missed that - thanks